module-cheatsheets/SCE-Seurat-cheatsheet.md

```diff
@@ -0,0 +1,104 @@
+
+
+```r
+# Convert Seurat object to SCE object, retaining only the 'RNA' experiment (assay)
+sce_object <- Seurat::as.SingleCellExperiment(seurat_obj, assay = "RNA")
+```
+
+Specifying `assay` is mostly useful if there are alternative experiments, for example from CITE-Seq data, present in the `Seurat` object that you do not want to retain during `SCE` conversion.
+
+## Converting from `SCE` to `Seurat`
+
+The following example code assumes you are starting with an `SCE` object called `sce_object`.
+
+The function `Seurat::as.Seurat()` can be used to convert an `SCE` object into a `Seurat` object and takes the following arguments:
+
+- The `SCE` object to convert
+- Optional named arguments with the following defaults:
+  - `counts = "counts"` specifies that the `SCE` object contains a `"counts"` assay of normalized counts that should be included during conversion.
+    - If there is no `"counts"` assay in the SCE object, set this argument as `counts = NULL` or rename accordingly, e.g. `counts = "whatever_assay_name_you_are_using"`.
+  - `data = "logcounts"` specifies that the `SCE` object contains a `"logcounts"` assay of normalized counts that should be included during conversion.
+    - If there is no `"logcounts"` assay in the SCE object, set this argument as `data = NULL` or rename accordingly, e.g. `data = "whatever_assay_name_you_are_using"`.
+  - `assay = NULL` specifies that, by default, all assays (experiments) will be converted. If there are multiple assays and you wish to only convert, for example, the `"RNA"` assay, set this argument as `assay = "RNA"`.
+  - `project = "SingleCellExperiment"` specifies that the `Seurat` object being created will have this associated project name. You can override this with any string of interest, e.g. `project = "sample_XYZ"`.
+
+
+```r
+# Convert SCE object to Seurat object, assuming both
+#  `counts` and  `logcounts` assays are present
+seurat_object <- Seurat::as.Seurat(sce_object)
+
+# Convert SCE object to Seurat object, where the SCE object
+#  contains a `counts` but not a `logcounts` assay
+seurat_object <- Seurat::as.Seurat(sce_object, data = NULL)
+```
+
+<div style="page-break-after: always;"></div>
+
+### Approaches from `ScPCA`
+
+In addition, this [documentation from the `ScPCA`](https://scpca.readthedocs.io/en/latest/faq.html#what-if-i-want-to-use-seurat-instead-of-bioconductor) introduces how to convert `SCE` objects to `Seurat` objects.
+Although this documentation was written for `ScPCA` datasets, the steps generally apply to any `SCE` object.
+It's worth noting that the example code provided at that link will only retain a single assay (raw `"counts"`) in the new `SCE` object, and it will not retain reduced dimension representations (e.g., PCA or UMAP).
+Therefore, this example code is mostly useful at the early stages of processing before you have performed normalization and calculated reduced dimensions.
```

Comment:
I know I suggested adding this, but I think it may be a bit out of date. I think the `Seurat::as.Seurat()` function should probably be preferred most of the time. My main thought for including this was for conversion to scanpy/scverse, but I think we need to revisit those as well. So maybe we don't include this until we update those.

Reply:
You mean this whole "approaches from ScPCA" section, right? Just confirming!

Reply:
Yes, the whole section can go for now!